Real_Masters_all/whitepet.xml

```diff
@@ -557,7 +561,7 @@
               <unittitle>Private Financial Journal <unitdate type="inclusive" normal="1868/1878">1868-1878</unitdate></unittitle>
             </did>
             <odd>
-              <p>(out. vol. #15)</p>
+              <p>(oversize volume no. 15)</p>
```

Comment:
It is throughout the rest of the EAD.

See also: https://github.com/devonproudfoot/vandura/blob/e8402ab2272baf218fbbf5e96aa18ea1c2bd560a/Real_Masters_all/whitepet.xml#L656

It does appear a little redundant, though, and is probably a holdover from when all of those containers were probably <physloc>Out. Vol</physloc>